flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/calcite/FlinkRexBuilder.java

```diff
@@ -102,10 +102,10 @@ public RexNode makeFieldAccess(RexNode expr, int i) {
      * @return Simple literal, or cast simple literal
      */
     @Override
-    public RexNode makeZeroLiteral(RelDataType type) {
+    public RexLiteral makeZeroLiteral(RelDataType type) {
```

Comment:
As a result of https://github.com/apache/calcite/commit/56a86a032ac05ed522846910eea6f884b31820e3#diff-3be99bddc7edf13dc8198da7425d7e97c97237e3561c263fd74903b4a42d8cd9R401